### ISSUE TITLE
larger batch size lead to worse accuracy

### ISSUE THREAD

**Issue by WIll-Xu35** [deepinsight/insightface#925]
Hi all,

I've been trying to train insightface from scratch for smaller input image size, 64x64. The sturcture is r50 and dataset is emore.

I first trained on my local workstation with batch size 128. The validation accuracy on agedb_30 is 0.96. Validation accuracy is 0.4, loss is around 6. And I tested it on my private dataset, had an accuracy of 0.996.

And then I trained it on a remote server using the same setting except batch size, this time I set it to 512 since the server has a better GPU. I also trained it using a single card. The validation accuracy on agedb_30 was 0.974, which is a little better as expected. Validation accuracy was 0.67, loss was 3. However, when I tested this model on my private dataset, the accuracy was only 0.96, 3.6% lower than the previous model.

The test codes are exactly the same. One possible reason might be the different cuda driver version I have locally and remotely (396 vs 418). And when running on remote server, I used docker for convenience. But I don't know how to eliminate this possibility except training again remotely using batch size 128.

Any idea what might caused this problem?

Thanks.
